flake-module.nix

```
@@ -15,12 +15,20 @@ in
             Project-level treefmt configuration
           '';
           type = types.submoduleWith {
-            modules = (import ./.).all-modules pkgs;
+            modules = (import ./.).all-modules pkgs ++ [{
+              options.enableAsFlakeFormatter = lib.mkOption {
```

```suggestion
              options.flakeFormatter = lib.mkOption {
```
Something a bit less verbose like that?